user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -51,7 +54,75 @@ function commonFunctions:isMediaApp()
   end
 
   return isMedia
+end
+
+function commonFunctions:waitForSDLStart(test)
+  local step = 100
+  local hmiPort = config.hmiPort
```

Comment:
@LitvinenkoIra assert that it is not nil or empty string. Do not forget that config is filled by people who can don't know about atf internals.